fli-iam/shanoir-ng
# shanoir-issue#2513: remove monoCenter

How to test:
Open a study with multiple center
Go to center tab
Edit > remove a center > update study > it should work

Another test:
Start import CT
At step 3, in center options, make sure it's only those of the study (check before and after removing a center from that study)

Another test:
Create a new study
Make sure a center is mandatory and that the "center" tab works well
